sort/countingsort.go

```diff
@@ -6,11 +6,13 @@
 
 package sort
 
-func Count(data []int) []int {
+import "github.com/TheAlgorithms/Go/constraints"
+
+func Count[T constraints.Number](data []int) []int {
```

Comment:
why isn't this
```
func Count[T constraints.Number](data []T) []T {
```
?

Reply:
Primarily because I missed it... Feel free to open a PR to rectify it.